src/maintenance/repositories/signing.md

```diff
@@ -0,0 +1,23 @@
+## Signing repositories
+
+Remote repositories are **must** be signed, for local repositories this is not
```

Comment:
typo "[repos] are must be"

Reply:
If we are here making changes, `local repositories do not need to be signed` is mildly more idiomatic